gc.c

```diff
@@ -1011,6 +1015,44 @@ asan_unlock_freelist(struct heap_page *page)
 
 #define GC_SWEEP_PAGES_FREEABLE_PER_STEP 3
 
+#define RVALUE_AGE_BITMAP_INDEX(n)  ((n) / (BITS_BITLENGTH / RVALUE_AGE_BIT_COUNT))
+#define RVALUE_AGE_BITMAP_OFFSET(n) ((n) % (BITS_BITLENGTH / RVALUE_AGE_BIT_COUNT) << 1)
```

Comment:
I think we need to make this vary based on the value of `RVALUE_AGE_BIT_COUNT`.

```suggestion
#define RVALUE_AGE_BITMAP_OFFSET(n) (((n) % (BITS_BITLENGTH / RVALUE_AGE_BIT_COUNT)) * RVALUE_AGE_BIT_COUNT)
```